modules/ROOT/pages/profiles-applications-overview.adoc

```diff
@@ -30,23 +29,16 @@ It should only be done to recover from an error temporarily. We strongly encoura
 
 === User
 
-By default, it gives access to the User navigation of Bonita Portal. It includes:
-
-* a xref:user-process-list.adoc[Process list]
-* a xref:portal-user-case-list.adoc[Case list]
-* a xref:user-task-list.adoc[Task list].
-This profile is also the only one mapped with Bonita Mobile Portal. +
-This profile is also mapped to the xref:user-application-overview.adoc[Bonita User Application], bringing the same value but with more customization options.
+By default, it gives access to the Bonita User Application.
 
 === Administrator
 
-By default, its gives access to the Administrator navigation of Bonita Portal. It includes:
+By default, its gives access to Bonita Administrator Application. It includes:
 
 * xref:_process-maintenance.adoc[Process Management] and xref:analytics.adoc[Analytics]
```

Comment:
xref:_process-maintenance.adoc[Process Management] and xref:analytics.adoc[Analytics] => xref:_process-maintenance.adoc[Process Management]

Reply:
no Analytics for release 2021.2

Reply:
Added